config/routes.rb

```diff
@@ -0,0 +1,24 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/merchants', to: 'merchants#index'
+
+  post '/merchants', to: 'merchants#create'
+  get '/merchants/new', to: 'merchants#new'
+
+  get '/merchants/:id/edit', to: 'merchants#edit'
+  patch '/merchants/:id', to: "merchants#update"
+
+  delete '/merchants/:id', to: 'merchants#delete'
+
+  get "/merchants/:merchant_id/items", to: 'items#index'
+
+  get '/merchants/:id', to: 'merchants#show'
+
+  get '/items', to: 'items#index'
+  get '/items/:id', to: 'items#show'
+
+  get '/merchants/:merchant_id/items/new', to: 'items#new' #new item form
+
+  post '/merchants/items', to: 'items#create'
```

Comment:
Because items are nested resources, you need to know the merchant's id to create the item, so this route should be something like

```
post '/merchants/merchant_id/items
```

If you pry into your items controller create action, you'll see you are trying to access params[:merchant_id], but it's nil because you didn't include it in the route. So you're not actually finding the right merchant. See the nested resources lesson plan if you need more help with this topic.